shared/ssa/qlpack.yml

```diff
@@ -1,4 +1,5 @@
 name: codeql/ssa
 version: 0.0.17-dev
 groups: shared
+warnOnImplicitThis: true
```

Comment:
Could we be a little more consistent with the ordering of the lines in these files? In some of these you add the line before `library: true` and in some you add it after. And for the language packs, I _think_ you've added these lines after dependencies, so maybe do that as well here (depending on whether the ordering in the language packs are consistent - I haven't checked).

Reply:
The ordering in language packs is not consistent, not even within each language. For language packs I added the `warnOnImplicitThis` option as the last line. I've this PR to add the option as the last line as well.